### ISSUE TITLE
quasar cli command "quasar c" conflict with clean and create

### ISSUE THREAD

**Issue by superoo7** [quasarframework/quasar#3229]
### Software version

Quasar:

```
Operating System	Darwin(18.2.0) - darwin/x64
NodeJs          	10.15.0

Global packages
  NPM           	6.6.0
  yarn          	1.13.0
  @quasar/cli   	1.0.0-beta.0
```

### What did you get as the error? What were you expecting?

When calling `quasar c` in development folder, I was expecting to run `quasar clean` but instead, it runs `quasar init`

### What steps did you take, to get the error?

```
# In Development folder, this run "quasar clean"
$ quasar c

# In empty folder, this run "quasar create"
$ quasar c
```

### How to fix

Refer to https://github.com/quasarframework/quasar/blob/866f3f385135529e8a6a0abc41cc6043ab448a3a/cli/bin/quasar#L11 and https://github.com/quasarframework/quasar/blob/5bc1ede861695bf72b534af5eca794974c604457/app/bin/quasar#L29

**Comment by rstoenescu** [comment]
Fix will be available in @quasar/cli v1.0.0-beta.1.
Thanks for also doing a PR!

**Comment by rstoenescu** [comment]
Released @quasar/cli v1.0.0-beta.1, containing the PR.
